seminar_registration_system/member_info_capture/views.py

```diff
@@ -1,25 +1,68 @@
 from django.shortcuts import render,redirect
-from .models import Member
+from member_info_capture.models import Member
+from django.urls import reverse
+from django.contrib import messages
+from django.http import HttpResponse
+from rest_framework import viewsets
+from .serializers import MemberSerializer
+
+class MemberViewSet(viewsets.ModelViewSet):
+    queryset = Member.objects.all()
+    serializer_class = MemberSerializer
+
+class MemberInfoCaptureViewSet(viewsets.ModelViewSet):
+    queryset = Member.objects.all()
+    serializer_class = MemberSerializer
```

Comment:
define views in the `views.py` file of my app to handle HTTP requests from the client